pkg/inventory/inventory-client_test.go

```diff
@@ -340,7 +340,7 @@ func TestReplace(t *testing.T) {
 			if err != nil {
 				t.Fatalf("unexpected error received: %s", err)
 			}
-			if !object.SetEquals(tc.localObjs, actualObjs) {
+			if !object.IsSetEqual(tc.localObjs, actualObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.